tensorly/tenalg/least_squares.py

```diff
@@ -0,0 +1,7 @@
+from .. import backend as T
+
+
+def solve_least_squares(A, B):
+    # solve ||B-AX||^2 (AX = B -> X = A^+ @ B), with A^+: pseudo inverse
+    X = T.pinv(A) @ B
```

Comment:
This is currently making the test fails as unfortunately other frameworks don't support it...
```suggestion
    X = T.dot(T.pinv(A), B)
```